src/book/garden-lattice/understanding.md

```diff
@@ -0,0 +1,113 @@
+> The presence of a `LICENSE` file is crucial for legal clarity, encourages the responsible use or distribution of the project, and is considered an indicator of project maturity{cite:p}`deekshitha_rsmm_2024`.
+
+Similar to how a garden's health depends on understanding the rules of nature and respecting boundaries, a project's sustainability relies on clear licensing that protects both the creators and users, fostering a culture of trust and collaboration.
+
+For further reading and examples of `LICENSE` files, see the following:
+
+- [OSF: Licensing](https://help.osf.io/article/148-licensing)
+- [GitHub: Licensing a repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository)
+- [Choosealicense.com](https://choosealicense.com/)
+- [SPDX License List](https://spdx.org/licenses/)
+
+## Project documentation
+
+Common documentation files like `README`'s, `CONTRIBUTING` guides, and `LICENSE` files are only a start towards more specific project information.
+Comprehensive project documentation is akin to a detailed gardener's notebook for a well-maintained project, illustrating how the project may be used and guiding philosophy.
+This includes in-depth explanations of the project's architecture, practical usage examples, API references, and development workflows.
+Oftentimes this type of documentation is provided through a "documentation website" or "docsite" to facilitate a user experience that includes a search bar, multimedia, and other HTML-enabled features.
+
+Such documentation should strive to go above and beyond to ensure that both novice and seasoned contributors can grasp the project's complexities and contribute effectively.
+"We … find that tools appear to be more widely used when software developers provide deeper documentation, …" {cite:p}`afiaz_evaluation_2023`.
+Just as a thriving garden benefits from meticulous care instructions and shared horticultural knowledge, a project flourishes when its documentation offers a clear and thorough guide to its inner workings, nurturing a collaborative and informed community.
+
+Project documentation often exists within a dedicated `docs` directory where the materials may be created and versioned distinctly from other code.
+Oftentimes this material will leverage a specific documentation tooling technology in alignment with the programming language(s) being used (for example, Python projects often leverage [Sphinx](https://www.sphinx-doc.org/en/master/)).
+These tools often increase the utility of the output by styling the material with pre-built HTML themes, automating application programming interface (API) documentation generation, and an ecosystem of plugins to help extend the capabilities of your documentation without writing new code.
```

Comment:
ah, API defined here. prob should be switched to first reference

Reply:
Thanks will do!